cmd/collector/app/handler/testdata/zipkin_proto_02.json

```diff
@@ -1,7 +1,7 @@
 {
   "spans": [
     {
-      "trace_id": "AAAAAAAAAAI=",
+      "trace_id":"QoVouqoJRn5CjmiKqglGfw==",
```

Comment:
(breaking 🛑) Zipkin Proto receiver requires 128bit trace ID, otherwise TraceID: has length 8 yet wanted length 16. Jaeger receiver did not have such limitation.